README.md

```diff
@@ -101,7 +101,19 @@ $ nodemon bin/www
 
 ## Static Code Analysis Tool 
 
-- JSLint
+- ESLint
```

Comment:
`###` for this section header, `####` for the `Install ESLint dependencies` and `Run ESLint Locally`

```markdown
### ESLint
```